querydsl-sql/src/main/java/com/querydsl/sql/Configuration.java

```diff
@@ -84,7 +75,7 @@ public Configuration(SQLTemplates templates) {
             javaTypeMapping.register(customType);
         }
         for (Map.Entry<SchemaAndTable, SchemaAndTable> entry : templates.getTableOverrides().entrySet()) {
-            schemaTables.put(entry.getKey(), entry.getValue());
+            nameMapping.registerTableOverride(entry.getKey(), entry.getValue());
```

Comment:
The only thing I see is since you make delegate methods to NameMapping, this should use the `registerTableOverride()` method for consistency.